### PR TITLE
Add capture_compatibility_patch_enabled as option in known_issues.md

### PR DESCRIPTION
**Pull request by tmaier**
Follow-up for #1650 and #1644

### CODE REVIEW

File: docs/known_issues.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ Some workarounds include:
   - Using FormBuilder with [Action View helpers](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html).
   - Using a FormBuilder overriding all field helpers to render a ViewComponent so each field can be customized individually (for example, [view_component-form](https://github.com/pantographe/view_component-form)).
   - Using a lightweight re-implementation of ViewComponent. For example, [Primer ViewComponents](https://github.com/primer/view_components) implemented [`ActsAsComponent`](https://github.com/primer/view_components/blob/main/lib/primer/forms/acts_as_component.rb) which is used in the context of `FormBuilder`.
+- Enable the capture compatibility patch with `config.view_component.capture_compatibility_patch_enabled = true`.
```
Comment (joelhawksley) [inline docs/known_issues.md:31]:
```suggestion
- Experimental: Enable the capture compatibility patch with `config.view_component.capture_compatibility_patch_enabled = true`.
```

File: docs/known_issues.md [hunk 2]
```diff
@@ -28,6 +28,8 @@ Some workarounds include:
...
+  Note that this patch is considered experimental.
```
Comment (joelhawksley) [inline docs/known_issues.md:32]:
```suggestion
```

**Review by joelhawksley — APPROVED** [review]

**Comment by Spone** [comment]
Maybe we can mention the patch before the workarounds, since it *really* solves the issue?

**Comment by tmaier** [comment]
Not that the CI pipeline fails due to an unrelated issue.

@Spone I thought about this. I decided to list it at the bottom for now, since it is experimental.

**Pull request merged**